includes/data-stores/class-wc-payment-token-data-store.php

```diff
@@ -305,7 +305,7 @@ public function get_users_default_token( $user_id ) {
 	 * Should contain the fields token_id, gateway_id, token, user_id, type, is_default.
 	 *
 	 * @since 3.0.0
-	 * @param id $token_id Token ID.
+	 * @param mixed $token_id Token ID.
```

Comment:
So these should be type `int` instead of `mixed` because that is what it is expecting (though not enforced). Same with the rest of the docblocks you've added `mixed` to. To stay consistent with other declarations seen here https://github.com/woocommerce/woocommerce/blob/master/includes/interfaces/class-wc-payment-token-data-store-interface.php#L44

Reply:
@roykho you're right, my bad.